models/base.py

```diff
@@ -13,7 +15,12 @@ def __init__(self, cv=5, n_iter=20):
 
     def fit(self, X, y):
         """Train on a training set and select optimal hyperparameters."""
-        # TODO: Impute missing predictors if necessary and set imputer here.
+        if X.isna().any():
```

Comment:
**Problem**
The truth value of a series is ambiguous.

**Fix**
@jirvin16

Reply:
Are you getting this error when running the code @ylzhouchris ? `X` should be a pandas dataframe here, and `X.isna().any()` should return a single boolean (this is the case with my data / pandas version). But please share the error if you're seeing it in your end (if you can share the data too that'd be ideal).